content/en/docs/tasks/security/authentication/mtls-migration/index.md

```diff
@@ -106,6 +106,15 @@ command terminated with exit code 56
 sleep.legacy to httpbin.bar: 200
 {{< /text >}}
 
+Optionally, you can verify that traffic is encrypted by running `tcpdump` in a proxy container, but only in an Istio installation with `values.global.proxy.privileged=true`. And you then can determine whether traffic is in plaintext
+or encrypted.
+
+{{< text bash >}}
+$ kubectl exec -nfoo $(kubectl get pod -nfoo -lapp=httpbin -ojsonpath={.items..metadata.name}) -c istio-proxy -it -- sudo tcpdump dst port 80  -A
+tcpdump: verbose output suppressed, use -v or -vv for full protocol decode
+listening on eth0, link-type EN10MB (Ethernet), capture size 262144 bytes
+{{< /text >}}
```

Comment:
[nit] May add a output snippet to highlight what information to look for.

Reply:
add description text.